compiler/luci/pass/src/RemoveRedundantTranspose.test.cpp

```diff
@@ -105,6 +105,95 @@ void create_redundunt_transpose(loco::Graph *g, const std::vector<int32_t> &perm
   auto graph_output = g->outputs()->create();
   output->index(graph_output->index());
 }
+/**
+ *  BEFORE
+ *             |
+ *       [CircleNode]       [CircleConst]
+ *                 \           /
+ *  [CircleConst] [CircleTranspose] [CircleConst]
+ *          \          / \              /
+ *     [CircleTranspose] [CircleTranspose]
+ *            |                |
+ *       [CircleNode]     [CircleNode]
+ *
```

Comment:
maybe add `|` after `[CircleNode]` items here and below

Reply:
I removed as I commented :(
https://github.com/Samsung/ONE/pull/5437#discussion_r549270909

Reply:
Sorry for the confision... I wanted to make things consistant... There was another comment https://github.com/Samsung/ONE/pull/5437#discussion_r549272751 :)

Reply:
then I  think remove all `|` under final node would be nice as it could be connect to any other node. what about this idea?

Reply:
After updating this, I will make a new PR for make a consistency on comment graph.